content/docs/hooks-reference.md

```diff
@@ -450,31 +451,31 @@ function FancyInput(props, ref) {
 FancyInput = forwardRef(FancyInput);
 ```
 
-In this example, a parent component that renders `<FancyInput ref={fancyInputRef} />` would be able to call `fancyInputRef.current.focus()`.
+Dalam contoh ini, sebuah komponen induk yang me-*render* `<FancyInput ref={fancyInputRef} />` akan dapat memanggil `fancyInputRef.current.focus()`.
 
 ### `useLayoutEffect` {#uselayouteffect}
 
-The signature is identical to `useEffect`, but it fires synchronously after all DOM mutations. Use this to read layout from the DOM and synchronously re-render. Updates scheduled inside `useLayoutEffect` will be flushed synchronously, before the browser has a chance to paint.
+Tanda ini identik dengan `useEffect`, tetapi diaktifkan secara *synchronous* setelah semua mutasi DOM. Gunakan ini untuk membaca tata letak dari DOM dan me-*render* ulang secara *synchronous*. Pembaruan yang dijadwalkan di dalam `useLayoutEffect` akan dipenuhi secara *synchronous*, sebelum browser memiliki kesempatan untuk menggambar.
 
-Prefer the standard `useEffect` when possible to avoid blocking visual updates.
+Pilih `useEffect` standar bila memungkinkan untuk menghindari pemblokiran pembaruan visual.
 
-> Tip
+> Kiat
```

Comment:
```suggestion
> Tip
```

We don't need to translate "tip", I think. We'll also have to check these blockquotes sometime in the future to make these blockquote heaedings consistent.